rep-2010.rst

```diff
@@ -234,7 +234,46 @@ Any existing tooling for aiding users in setting up ROS security should consider
 Command line interface
 ----------------------
 
-TODO: Consider proposing an interface for a CLI here. Although it seems a bit tangent to the core feature, I think it is important to the proposal since it makes it so people can immediately start using the feature.
+Suppose we have a service named ``/add_two_ints`` of type ``example_interfaces/srv/AddTwoInts``.
+We can have a tool like ``ros2 service echo /add_two_ints`` that internally subscribes to the two hidden topics ``example_interfaces/srv/AddTwoInts_RequestEvent``, ``example_interfaces/srv/AddTwoInts_ResponseEvent`` and echoes them. It should also support additional command line parameters used for topics, like ``--filter, --no-arr, --no-str, --include-msg-info``, etc.
+
+For example,
+.. code-block::
+
+   $ ros2 service echo /add_two_ints
+   -----------------------
+   request_type: REQUEST_SENT
+   stamp: 1.00
+   client_id: 1234
+   sequence_number: 1
+   request:
+      a: 1
+      b: 2
+   -----------------------
+   request_type: REQUEST_RECEIVED
+   stamp: 1.10
+   client_id: 1235
+   sequence_number: 1
+   request:
+      a: 1
+      b: 2
+   -----------------------
+   request_type: RESPONSE_SENT
+   stamp: 1.20
+   client_id: 1235
+   sequence_number: 2
+   request:
+      sum: 3
+   -----------------------
+   request_type: RESPONSE_RECEIVED
+   stamp: 1.30
+   client_id: 1234
+   sequence_number: 2
+   request:
+      sum: 3
+   -----------------------
+
+Additional parameters could be included like ``--response-time``, which should print the delay between the time the client sent the request and the server sent the response, or ``--exculde-msg-content`` to just echo the events and not the content of messages.
```

Comment:
"response_time" seems overloaded here, it might be confused with the time a response was received, for example. Maybe "delay" is a better term? Also, I'm not sure what the output would look like in this case. Would delay value be echo'd as part of the response received event only, or something else? IMO, it might be best to be vague here and not mention specific options. These are subject to change anyways over time.

Reply:
Edited the section to remove specific command line options, moved the example to "feature progress" section.